ejb3/src/main/resources/schema/jboss-ejb-client_1_4.xsd

```diff
@@ -0,0 +1,337 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
FYI it is easier to review changes if a new schema is added in it's own commit followed by the changes applied for the RFE, otherwise the only way to see the changes is by manually diffing the files.

Reply:
fixed